### DISCUSSION TITLE
Multiple container items referring to the same `canvas.Image` only render once

### DISCUSSION THREAD

**Discussion by srackham** [fyne-io/fyne#4442] [Q&A] [answered]
The following code only renders one image item, the other one is blank (Fyne v2.4.2):

```go
func main() {
	a := app.New()
	w := a.NewWindow("Test")

	img := canvas.NewImageFromFile("testdata/image-01.jpg")
	img.SetMinSize(fyne.NewSize(320, 240))

	c := container.New(layout.NewVBoxLayout(), img, img)
	w.SetContent(c)

	w.ShowAndRun()
}
```
![screenshot](https://github.com/fyne-io/fyne/assets/674468/bda57bb7-1f12-4e6a-980f-f7ecea84d72b)

I made a copy of the image and passed it as follows:

```go
img2 := *img
c := container.New(layout.NewVBoxLayout(), img, &img2)
```
This seems to work but I got the following warning:
`assignment copies lock value to img2: fyne.io/fyne/v2/canvas.Image contains fyne.io/fyne/v2/canvas.baseObject contains sync.RWMutex`

Why is this, and what would be the best work-around?

**Comment by andydotxyz** [comment, marked as answer]
Objects can only be visible in one place at a time, if you want to display the same image twice you will need two `canvas.Image` objects referring to the same file/resource/pixels.

**Reply by srackham** [reply]
Thankyou.
